frontend/tailwind.config.cjs

```diff
@@ -27,14 +112,130 @@ const config = {
 			},
 			transitionProperty: {
 				'height': 'height'
+			},
+			fontSize: {
+				'2xs': '0.7rem'
 			}
 		}
 	},
 
 	plugins: [
 		require('@tailwindcss/forms'),
 		require('@tailwindcss/typography'),
-		require('flowbite/plugin')
+		require('flowbite/plugin'),
+		plugin(({ addBase, addComponents, addUtilities, theme }) => {
+			addBase({
+				'html': {
+					overflowY: 'auto'
+				},
+				'h1': {
+					fontSize: theme('fontSize.2xl'),
+					color: theme('colors.gray.700')
+				},
+				'h1': {
+					fontSize: theme('fontSize.xl'),
+					color: theme('colors.blue.500')
+				},
+				'a': {
+					color: theme('colors.blue.500')
+				},
+				'input,input[type="text"],input[type="email"],input[type="url"],input[type="password"],input[type="number"],input[type="date"],input[type="datetime-local"],input[type="month"],input[type="search"],input[type="tel"],input[type="time"],input[type="week"],textarea,select': {
```

Comment:
@adam-kov Do you know whether we can customise inputs down the line with custom TW without using important?

Reply:
Yes, this rule targets only by element names. Class targeting has higher specificity score, so those classes will ovveride these styles.